pkg/chain/chain.go

```diff
@@ -21,20 +21,20 @@ type Handle interface {
 	// BlockCounter returns a block counter.
 	BlockCounter() chain.BlockCounter
 
-	BondedECDSAKeepFactory
-	BondedECDSAKeep
+	KeepFactory
+	Keep
 }
 
-// BondedECDSAKeepFactory is an interface that provides ability to interact with
-// BondedECDSAKeepFactory ethereum contracts.
-type BondedECDSAKeepFactory interface {
+// KeepFactory is an interface that provides ability to interact with
+// keep factory ethereum contracts.
+type KeepFactory interface {
 	// RegisterAsMemberCandidate registers client as a candidate to be selected
 	// to a keep.
 	RegisterAsMemberCandidate(application common.Address) error
 
-	// OnBondedECDSAKeepCreated installs a callback that is invoked when an
+	// OnKeepCreated installs a callback that is invoked when an
 	// on-chain notification of a new bonded ECDSA keep creation is seen.
-	OnBondedECDSAKeepCreated(
+	OnKeepCreated(
 		handler func(event *BondedECDSAKeepCreatedEvent),
 	) subscription.EventSubscription
 
```

Comment:
I'm not sure about these renames. The existing contract is still named `BondedECDSAKeepFactory` and event is still `BondedECDSAKeepCreatedEvent`.